README.md

```diff
@@ -1,169 +1,13 @@
 # Intel® Platform Monitoring Technology Backports
 
-This repo holds backports of the pmt driver.
+## Deprecated
 
-The driver is provided as a DKMS module targeting distinct
-versions of various distros. Each backport is on a topic 
-branch. Current support status is documented on the topic branch.
+To provide unified kmd drivers backports needed for intel graphics platforms.
```

Comment:
KMD - make it caps

Reply:
To provide unified KMD backports needed for  Intel Graphics Platforms, backport of ...